src/index.ts

```diff
@@ -1,7 +1,8 @@
-export * from "./Button/Button";
-export * from "./Button/ButtonPrimary";
-export * from "./Content/Content";
-export * from "./Dialog/DialogFullScreen";
-export * from "./Input/RadioCards";
+export * from './Button/Button';
+export * from './Button/ButtonPrimary';
+export * from './Content/Content';
+export * from './Dialog/DialogFullScreen';
+export * from './Input/RadioCards';
```

Comment:
I think prettier didn't run for this file. It only runs for `--staged` files for the git hook. Can you just run
```bash
$ yarn pretty-quick
```